app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -559,6 +567,10 @@ def rbac_user_delete_restriction?(user)
     ["admin", session[:userid]].include?(user.userid)
   end
 
+  def rbac_user_copy_restriction?(user)
+    user.userid == "admin"
```

Comment:
@lpichler The parameter for the function comes from checked item in the list of users:

```ruby
user = User.find(from_cid(params[:miq_grid_checks]))
```

the method you suggest checks if current user is admin:

```ruby
current_user.try(:super_admin_user?)
```

I don't think that's the same. Please correct me if I'm wrong.